github-actions/sdk-readme-generator/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @actions/actions-runtime
```

Comment:
github is showing that this file has errors
<img width="674" alt="Screen Shot 2022-07-15 at 9 14 06 AM" src="https://user-images.githubusercontent.com/11823378/179264265-278e1443-b596-421b-a3fd-5c8e654ab2b3.png">